cmd/registry/cmd/upload/csv.go

```diff
@@ -44,37 +43,33 @@ func csvCommand() *cobra.Command {
 		Use:   "csv file --project-id=value [--delimiter=value]",
```

Comment:
We shouldn't include the flags in the `Use:` as they are listed automatically. (Also, `--project-id` is deprecated.)

Reply:
thanks, I'll take those out